source/strings.tex

```diff
@@ -4174,6 +4180,36 @@
 and initializing \tcode{size_} with \tcode{len}.
 \end{itemdescr}
 
+\indexlibraryctor{basic_string_view}%
+\begin{itemdecl}
+template<class It, class End>
+  constexpr basic_string_view(It begin, End end);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+\begin{itemize}
+\item \tcode{It} satisfies \tcode{contiguous_iterator},
```

Comment:
```suggestion
\item \tcode{It} satisfies \libconcept{contiguous_iterator},
```
[As done in the similar change to `span`.](https://github.com/cplusplus/draft/pull/3456/files#diff-0fb8725b6f959c00f8cef02d54e8c125R10736)

Reply:
Yes, please. This adds nice entries to the "index of concept names".